jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/ButtonToggleTest.java

```diff
@@ -0,0 +1,129 @@
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.buttonToggleSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class ButtonToggleTest extends TestsInit {
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+    }
+
+    @Test
+    public void basicButtonToggleDisplayedTest() {
+        buttonToggleSection.basicButtonToggleBold.isDisplayed();
+        buttonToggleSection.basicButtonToggleItalic.isDisplayed();
+        buttonToggleSection.basicButtonToggleUnderline.isDisplayed();
+    }
+
+    @Test
+    public void basicButtonToggleEnabledTest() {
+        buttonToggleSection.basicButtonToggleBold.is().enabled();
+        buttonToggleSection.basicButtonToggleItalic.is().enabled();
+        buttonToggleSection.basicButtonToggleUnderline.is().enabled();
+    }
+
+    @Test
+    public void basicBoldButtonToggleTest() {
+        String pressedAttribute = "aria-pressed";
+        String buttonIsPressed = "true";
+        String buttonIsNotPressed = "false";
+        buttonToggleSection.basicButtonToggleBold.click();
+        buttonToggleSection.basicButtonToggleBold.has().attr(pressedAttribute, buttonIsPressed);
+        buttonToggleSection.basicButtonToggleBold.click();
+        buttonToggleSection.basicButtonToggleBold.has().attr(pressedAttribute, buttonIsNotPressed);
+    }
+
+    @Test
+    public void basicItalicButtonToggleTest() {
+        String pressedAttribute = "aria-pressed";
+        String buttonIsPressed = "true";
+        String buttonIsNotPressed = "false";
+        buttonToggleSection.basicButtonToggleItalic.click();
+        buttonToggleSection.basicButtonToggleItalic.has().attr(pressedAttribute, buttonIsPressed);
+        buttonToggleSection.basicButtonToggleItalic.click();
+        buttonToggleSection.basicButtonToggleItalic.has().attr(pressedAttribute, buttonIsNotPressed);
+    }
+
+    @Test
+    public void basicUnderlineButtonToggleTest() {
+        String pressedAttribute = "aria-pressed";
+        String buttonIsPressed = "true";
+        String buttonIsNotPressed = "false";
+        buttonToggleSection.basicButtonToggleUnderline.click();
+        buttonToggleSection.basicButtonToggleUnderline.has().attr(pressedAttribute, buttonIsPressed);
+        buttonToggleSection.basicButtonToggleUnderline.click();
+        buttonToggleSection.basicButtonToggleUnderline.has().attr(pressedAttribute, buttonIsNotPressed);
+    }
+
+    @Test
+    public void basicButtonToggleNamesTest() {
+        buttonToggleSection.basicButtonToggleBold.has().text("Bold");
+        buttonToggleSection.basicButtonToggleItalic.has().text("Italic");
+        buttonToggleSection.basicButtonToggleUnderline.has().text("Underline");
+    }
+
+    @Test
+    public void pressAllBasicButtonTogglesTest() {
```

Comment:
See the allure report.
Could you find out the reason why this method does not have no any reports (logs).
Please try to solve this.

Reply:
I'm not sure about that, maybe there are no reports because all steps of this test were logged before (in previous tests)?